x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/shared_components/form_row.scss

```diff
@@ -0,0 +1,3 @@
+.lnsInnerIndexPattern__inlineRow {
+  min-width: 96px;
```

Comment:
Again, I hadn't considered the "Percentile ranks value" input when putting together the mockups. I think we'll need to increase the `min-width` here in order to ensure the prepended labels are the same width (or alternatively come up with a shorter label than "Percentile ranks value").

![image](https://user-images.githubusercontent.com/3884767/177376657-78c02080-5a1e-42a4-83f8-216a488a40ce.png)

```suggestion
  min-width: 144px;
```

Reply:
That unfortunately has the other effect when we don't use percentile ranks - it takes too much space... 

<img width="351" alt="Screenshot 2022-07-05 at 23 15 22" src="https://user-images.githubusercontent.com/4283304/177417849-4d02713c-66aa-4eda-99a1-ee3b43fa13c1.png">

Reply:
Yeah. What about a shorter name for "Percentile ranks value"? For example, would simply "Percentile rank" work? Then we could adjust the label min-width to be equal to the largest of the labels.